### PR TITLE
Tests and other changes

### PR DESCRIPTION
**Pull request by CodingIsBliss**
Tests and other changes

### CODE REVIEW

File: src/BuildScriptGenerator.Common/SdkStorageConstants.cs [hunk 1]
```diff
@@ -6,22 +6,15 @@ public static class SdkStorageConstants
...
-        public const string SandboxSdkStorageBaseUrl = "https://oryxsdkssandbox.blob.core.windows.net";
-        public const string PrivateSdkStorageBaseUrl = "https://oryxsdksprivate.blob.core.windows.net";
         public const string ProdSdkStorageBaseUrl = "https://oryxsdksprod.blob.core.windows.net";
         public const string ProdBackupSdkStorageBaseUrl = "https://oryxsdksprodbackup.blob.core.windows.net";
         public const string ProdSdkCdnStorageBaseUrl = "https://oryx-cdn.microsoft.io";
         public const string DefaultVersionFileName = "defaultVersion.txt";
         public const string DefaultVersionFilePrefix = "defaultVersion";
         public const string DefaultVersionFileType = "txt";
         public const string VersionsToBuildFileName = "versionsToBuild.txt";
-        public const string ContainerMetadataUrlFormat = "{0}/{1}?restype=container&comp=list&include=metadata&marker={2}&{3}";
```
Comment (sarsharma) [inline src/BuildScriptGenerator.Common/SdkStorageConstants.cs:24]: what argument are we removing here?

Comment (CodingIsBliss) [reply src/BuildScriptGenerator.Common/SdkStorageConstants.cs:24]: sas token

File: tests/Oryx.BuildImage.Tests/CommandTests/PrepareEnvironmentCommandTest.cs
Comment (sarsharma) [inline tests/Oryx.BuildImage.Tests/CommandTests/PrepareEnvironmentCommandTest.cs:1]: lets re-enable all these tests?

File: tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs [hunk 2]
```diff
@@ -44,18 +44,18 @@ public DotNetCoreDynamicInstallTest(ITestOutputHelper output) : base(output)
...
-        {
-            BuildsApplication_ByDynamicallyInstallingSDKs(
-                appName, runtimeVersion, _restrictedPermissionsImageHelper.GetGitHubActionsBuildImage());
-        }
+        // [Theory, Trait("category", "githubactions")]
+        // [InlineData(NetCoreApp21WebApp, "2.1")]
+        // [InlineData(NetCoreApp31MvcApp, "3.1")]
+        // [InlineData(NetCoreApp50MvcApp, "5.0")]
+        // [InlineData(NetCore7PreviewMvcApp, "7.0")]
+        // public void BuildsApplication_ByDynamicallyInstallingSDKs_GithubActions(
```
Comment (sarsharma) [inline tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs:52]: re-enable this?

File: tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs
Comment (sarsharma) [inline tests/Oryx.BuildImage.Tests/Node/NodeDynamicInstallationTest.cs:1]: update these tests to buster/bullseye and enable them?

File: tests/Oryx.Tests.Common/TestValueGenerator.cs [hunk 3]
```diff
@@ -12,10 +12,9 @@ public static class TestValueGenerator
     {
         private readonly static List<(string Version, string OsType)> NodeVersions = new List<(string, string)>
         {
-            ("14", ImageTestHelperConstants.OsTypeDebianBuster),
```
Comment (sarsharma) [inline tests/Oryx.Tests.Common/TestValueGenerator.cs:15]: let's revert this

File: tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs
Comment (sarsharma) [inline tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs:1]: let's re-enable these as well?

**Review by sarsharma — APPROVED** [review]

**Review by kumaraksh1 — APPROVED** [review]
LGTM !!

**Pull request merged**